### ISSUE TITLE
Didn't extract any relationships with gpt_4o_mini_complete, working with gpt_4o_mini_complete

### ISSUE THREAD

**Issue by rcoundon** [HKUDS/LightRAG#301]
When using gpt_4o_complete to create the knowledge graph I'm seeing the warning:
`WARNING:lightrag:Didn't extract any relationships, maybe your LLM is not working`

Instantiation looks like this:
```py
rag = LightRAG(
            working_dir=working_dir,
            llm_model_func=gpt_4o_complete, 
            graph_storage="Neo4JStorage",
            log_level="INFO",
        )
```

I don't see the same warning when using gpt_4o_mini_complete

The app is creating a knowledge graph for chunks of some markdown files (originally converted from PDF)
Any thoughts on what could be causing this?

**Comment by rcoundon** [comment]
I'm not sure it's related, but when using the mini there is still a warning but it's seemingly unrelated:


`WARNING:neo4j.notifications:Received notification from DBMS server: {severity: WARNING} {code: Neo.ClientNotification.Statement.UnknownLabelWarning} {category: UNRECOGNIZED} {title: The provided label is not in the database.} {description: One of the labels in your query is not available in the database, make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing label name is: NCRNCSNCT)} {position: line: 1, column: 10, offset: 9} for query: 'MATCH (n:`NCRNCSNCT`) RETURN n'`

I'm not actually issuing queries at this point, just creating the KGs for my docs.

**Comment by LarFii** [comment]
You should check the output from the LLM during the extraction process. You can directly review the cache files to see if the output is as expected. This can help determine whether the issue lies in the LLM's response or elsewhere in the pipeline.

**Comment by rcoundon** [comment]
> You should check the output from the LLM during the extraction process. You can directly review the cache files to see if the output is as expected. This can help determine whether the issue lies in the LLM's response or elsewhere in the pipeline.

Ok, thanks, I'll take a look and report back

**Comment by LarFii** [comment]
Yes, I think that is the root cause of the issue. Modifying the entity types in `prompt.py` might help resolve it.

**Comment by GaryDean** [comment]
I came across this issue while processing a large number of files. The warning message `WARNING:lightrag:Didn't extract any relationships, maybe your LLM is not working` only ever occurred with _very small files_ (<50 bytes) that, in fact, contained no relevant information.  Solution was to remove such files.
